enhancements/hypershift/topology-aware-scheduling/topology-aware-scheduling.md

```diff
@@ -0,0 +1,256 @@
+---
+title: topology-aware-scheduling
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@ffromani"
+  - "@csrwng"
+  - "@sjenning"
+approvers: 
+  - "@jparrill"
+  - "@alvaroaleman"
```

Comment:
Please remove Alvaro since he's not at Red Hat

Reply:
Done. Thanks